app/wama.web/Controllers/CheckInController.cs

```diff
@@ -76,10 +87,15 @@ public IActionResult WaiverNotSigned()
         {
             var userAccount = await _UserAccountService.GetUserAccountAsync(memberId);
 
-            string userName = userAccount.FirstName + " " + userAccount.LastName;
-
-            if (signerName == null || !signerName.Equals(userName))
+            string userName = null;
+            if (userAccount != null)
             {
+                userName = userAccount.FirstName + " " + userAccount.LastName;
+            }
+            
+
+            if (signerName == null || (userName != null && !signerName.ToLower().Equals(userName.ToLower())))
```

Comment:
This looks good, but what about something like this.

(string.IsNullOrWhiteSpace(signerName) || string.IsNullOrWhiteSpace(userName) || !signerName.Equals(userName, StringComparison.OrdinalIgnoreCase))